packages/teams-js/src/private/nestedAppAuth/nestedAppAuthBridge.ts

```diff
@@ -1,5 +1,8 @@
 import { v4 as generateUUID } from 'uuid';
```

Comment:
Please verify request with tag in Orange test app to make sure it works on host. Make sure host can receive message with tag and process it correctly (generally speaking it shouldn't break anything)

Reply:
Already tested.